src/main/java/org/hibernate/search/develocity/plugins/SurefireConfiguredPlugin.java

```diff
@@ -26,14 +33,56 @@ protected void configureTest(GoalMetadataProvider.Context context) {
 			dependsOnConfigurableJavaExecutable( inputs, context, "jvm", isSkipped( context ),
 					JavaVersions::forJavaExecutable );
 		} );
+
+		configureEnvironmentVariables(context);
+	}
+
+	// Develocity handles environment variables as a big blob by default,
+	// which won't work if some variables point to absolute paths.
+	private void configureEnvironmentVariables(GoalMetadataProvider.Context context) {
+		context.metadata().inputs( inputs -> {
+			// First, override the property to disable handling of environment variables as a blob.
+			// NOTE: ignoring with inputs.ignore( "environmentVariables" ) doesn't work for some reason:
+			// we end up with the goal being marked as "not cacheable"
+			// because "properties were declared both as input and ignored: [environmentVariables]"
+			// NOTE: we get the same result with context.nested( "environmentVariables" ),
+			// which is why we don't use that.
+			inputs.property( SUREFIRE_ENVIRONMENT_VARIABLES, "IGNORED" );
```

Comment:
This is the workaround @jprinet suggested as an alternative to `inputs.ignore`, and it works!